core/src/test/java/org/elasticsearch/index/query/MissingQueryBuilderTest.java

```diff
@@ -68,96 +63,8 @@ private String getRandomFieldName() {
     }
 
     @Override
-    protected Query doCreateExpectedQuery(MissingQueryBuilder queryBuilder, QueryParseContext context) throws IOException {
-        final boolean existence = queryBuilder.existence();
-        final boolean nullValue = queryBuilder.nullValue();
-        String fieldPattern = queryBuilder.fieldPattern();
+    protected void doAssertLuceneQuery(MissingQueryBuilder queryBuilder, Query query, QueryParseContext context) throws IOException {
```

Comment:
right, I think you can clearly ee that I gave up on this query... I can look again but it becomes too complex to test anything here... I also want to try to stay away from mappings etc.